prime-router/src/main/kotlin/SettingsProvider.kt

```diff
@@ -59,4 +59,7 @@ enum class Topic(val json_val: String) {
 
     @JsonProperty("test")
     TEST("test"),
+
+    @JsonProperty("full-elr-test")
+    FULL_ELR_TEST("full-elr-test"),
```

Comment:
What is this used for?

Reply:
I needed a way to differentiate between covid tests and UP tests so that it would pass the if statement on line 138 of Translator.kt only when I wanted it to without using a production topic.

Reply:
We don't want any new code for the legacy pipeline.